src/Controller/Simulation.java

```diff
@@ -467,6 +468,21 @@ public String toggleRunning() {
         this.running = true;
         return "Simulation ON";
     }
+    
+    /**
+     * Toggles the Auto Mode of the light
+     * 
+     * @return console msg indicating Light Auto mode state.
+     */
+    public String toggleLight() {
+    	LightAuto = true;
```

Comment:
this line should be in the constructor of the SImulation class.  Rn it sets lightauto to true every time this method is called, so light auto mode is always turned ON, never OFF.

Reply:
My mistake! Should be fixed now!